pkg/updatestatus/nodeinformer.go

```diff
@@ -601,17 +598,22 @@ func nodeInformerControllerQueueKeys(object runtime.Object) []string {
 		return nil
 	}
 	switch o := object.(type) {
+	case *corev1.Event:
+		if o.Name != eventNameReconcileAllNodes {
+			panic(fmt.Sprintf("USC :: Unknown object %s with type: %T", o.Name, object))
+		}
+		return []string{queueKeyFor(eventKindName, o.Name)}
```

Comment:
I don't think this code is actually necessary - `nodeInformerControllerQueueKeys` is only used in `WithInformersQueueKeysFunc` so that the controller library can translate Objects into queue key, but we should never need that for our synthetic keys - we put the keys into the queue directly and we make them ourselves with `queueKeyFor`

Reply:
Nice. I thought that even for the synthetic keys, it would still go through the function.
That was why I took all the troubles finding a k8s resource that sounds very general.
Of course, your explanation makes more sense.

Reply:
Removed.